zipline/utils/calendars/trading_calendar.py

```diff
@@ -684,17 +685,22 @@ def minute_to_session_label(self, dt, direction="next"):
             The label of the containing session.
         """
 
-        idx = searchsorted(self.market_closes_nanos, dt.value)
+        if type(dt) is pd.Timestamp:
```

Comment:
Yea, I recommend `isinstance` one way or the other.  Then it's more flexible with subclasses.  

Take a look at the `coerce` preprocessor - it uses `isinstance` [internally] (https://github.com/quantopian/zipline/blob/bfa3e6f153698df8dd04334012aeacde9f3912c9/zipline/utils/input_validation.py#L575-L575) to decide whether to run the coercion function or pass through the original arg.